frame/system/src/lib.rs

```diff
@@ -609,11 +301,15 @@ decl_module! {
 		/// - 1 event.
 		/// The weight of this function is dependent on the runtime. We will treat this as a full block.
 		/// # </weight>
-		#[weight = (T::BlockWeights::get().max_block, DispatchClass::Operational)]
-		pub fn set_code_without_checks(origin, code: Vec<u8>) {
+		#[pallet::weight((T::BlockWeights::get().max_block, DispatchClass::Operational))]
+		pub fn set_code_without_checks(
+			origin: OriginFor<T>,
+			code: Vec<u8>
```

Comment:
I don't mind, but this is the style_guide :-)
```suggestion
			code: Vec<u8>,
```